content/en/blog/porterlb-high-availability-in-layer-2-mode.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "PorterLB: High Availability Mechanisms in Layer 2 Mode"
+linkTitle: "PorterLB: High Availability Mechanisms in Layer 2 Mode"
+description: How PorterLB ensures high availability in Layer 2 mode.
+keywords: PorterLB, Bare Metal, Kubernetes, High Availability, Layer 2 
+author: Patrick
+date: 2021-03-30
+weight: 99998
+---
+
+PorterLB makes it possible for users to expose Services in bare-metal Kubernetes environments. Currently, PorterLB supports the BGP mode and the Layer 2 mode, which use BGP and ARP/NDP respectively to expose Services.
+
+Generally, the BGP mode is recommended because it allows you to build a high availability system free of failover interruptions and bandwidth bottlenecks. However, BGP may be unavailable in certain systems because of security requirements or because the router does not support BGP. In this case, you can use PorterLB in Layer 2 mode to achieve similar functionality.
+
+Though the Layer 2 mode does not provide the same high availability as the BGP mode, it does implement certain mechanisms to ensure that PorterLB can still function as long as the Kubernetes cluster is not entirely down.
+
+In this article, I am going to discuss the high availability mechanisms of PorterLB in Layer 2 mode. The following scenarios will be examined:
+
+* Scenario 1: Only one PorterLB replica is deployed and the node selected by PorterLB is down.
+* Scenario 2: Only one PorterLB replica is deployed and the node where PorterLB is deployed is down.
+* Scenario 3: Multiple PorterLB replicas are deployed and one of the nodes that contain PorterLB replicas is down.
+
+## Scenario 1
```

Comment:
Could you add a brief description for each scenario? The sub-headlines are critical for the Google search engine, considering how people find these scenarios and this blog in searching keywords.